php/class-photonfill.php

```diff
@@ -611,17 +623,10 @@ public function get_lazyload_image( $attachment_id, $size = 'full', $attr = arra
 				}
 				$attr['class'][] = 'lazyload';
 			}
-			$srcset = $this->get_responsive_image_attribute( $attachment_id, $size, 'data-srcset' );
-			$sources = explode( ',', $srcset );
-			$src = explode( ' ', $sources[0] );
-			return sprintf(
-				'<img data-sizes="auto" data-src="%s" data-srcset="%s" class="%s" %s %s>',
-				esc_url( $src[0] ),
-				esc_attr( $srcset ),
-				esc_attr( $this->get_image_classes( $attr['class'], $attachment_id, $size ) ),
-				( ! empty( $attr['alt'] ) ) ? ' alt="' . esc_attr( $attr['alt'] ) . '"' : '',
-				( ! empty( $attr['style'] ) ) ? ' style="' . esc_attr( $attr['style'] ) . '"' : ''
-			);
+
+			$attr['class'] =  $this->get_image_classes( $attr['class'], $attachment_id, $size );
+
+			return wp_get_attachment_image( $attachment_id, $size, false, $attr );
```

Comment:
same issue here. It might be best to create our own get_attachment_image which allows width, height and src to be unset.